### ISSUE TITLE
Content not replaced/hydrated when using Embroider

### ISSUE THREAD

**Issue by josemarluedke** [ember-fastboot/ember-cli-fastboot#785]
Adding Embroider to a FastBoot site causes the content to not be cleared, instead the content is rendered twice.

The issue can be seen in this [PR](https://github.com/josemarluedke/docfy/pull/53).

Note, I have `EXPERIMENTAL_RENDER_MODE_SERIALIZE` enabled, but it does not affect the bug if I disable that.

**Comment by kiwiupover** [comment]
@josemarluedke I got your PR running and every seems fine to me. 

Why do you think it is rendering the content twice?

**Comment by kiwiupover** [comment]
@josemarluedke ping are you stilling having issues here?

**Comment by josemarluedke** [comment]
Yes, I'm still having this issue. It's weird you were not able to see in that PR. The entire site is rendered twice. I can post a screenshot when I am back to the computer.

**Comment by kiwiupover** [comment]
What do you mean by rendered twice?

**Comment by josemarluedke** [comment]
Here is an image showing that the HTML generated in FastBoot was not removed when the client version executed, the client version just prepended the content, leaving the HTML from the FastBoot and adding the HTML for the client. That makes it show the entire site twice on the same page. 

<img width="1716" alt="Screen Shot 2020-10-02 at 8 40 27 AM" src="https://user-images.githubusercontent.com/230476/94942535-49335c00-048b-11eb-9d5c-df28980472f2.png">

**Comment by kiwiupover** [comment]
Interesting. I will have a further look in the next few days.

**Comment by rahulk94** [comment]
@kiwiupover @josemarluedke did anything ever come out of this investigation?
